File: clojure/FizzBuzz.clj
(defn fizzbuzz [end]
    (doseq
        [s (map
            (fn [n]
                (cond
                    (zero? (mod n 15)) "FizzBuzz"
                    (zero? (mod n 3)) "Fizz"
                    (zero? (mod n 5)) "Buzz"
                    :else n
                )
            )
            (range 1 (+ end 1))
        )]
        (println s)
    )
)

(fizzbuzz 100)

File: javascript/FizzBuzz.js
function fizzBuzz(end) {
    for (var i = 1; i <= end; i++) {
        console.info(new Array(Number(i % 3 === 0) + 1).join("Fizz") + new Array(Number(i % 5 === 0) + 1).join("Buzz") || i);
    }
}

fizzBuzz(100)